panel/tests/chat/test_message.py

```diff
@@ -292,6 +293,7 @@ def test_serialize_svg_embed(self):
             message = ChatMessage(SVG(buf))
         assert message.serialize() == "SVG"
 
+    @pytest.mark.skipif(os.name == 'nt', reason="Wrong Windows path")
```

Comment:
@ahuang11 This is an actual bug, see https://github.com/holoviz/panel/actions/runs/6892917266/job/18751302119?pr=5883#step:7:9865

``` python

FAILED panel/tests/chat/test_message.py::TestChatMessage::test_serialize_audio - assert "Audio='D:\\\...s\\\\mp3.mp3'" == "Audio='D:\\a...ets\\mp3.mp3'"

  - Audio='D:\a\panel\panel\panel\tests\pane\assets\mp3.mp3'

  ?                   -----

  + Audio='D:\\a\\panel\\panel\\panel\\tests\\pane\\assets\\mp3.mp3'

  ?           + +              ++++++++      +    +       +

= 1 failed, 2092 passed, 170 skipped, 1 xfailed, 135 warnings in 136.04s (0:02:16) =
```

Reply:
@ahuang11 Did you ever get a chance to look at this?